recipe_scrapers/donalskehan.py

```diff
@@ -0,0 +1,79 @@
+from ._abstract import AbstractScraper
+from ._exceptions import StaticValueException
+from ._utils import get_minutes, get_yields
+
+
+class DonalSkehan(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "donalskehan.com"
+
+    def author(self):
+        image_element = self.soup.find("img", class_="img-fluid")
+        return image_element.get("alt") if image_element else "Donal Skehan"
```

Comment:
There doesn't seem to be any test coverage excercising the guest/other author case here; could we add some?

Reply:
There aren't any i could find. I removed the `else`

Reply:
Ok, thanks! - this could be a possible candidate for an idea I've described in #1245.